README.md

```diff
@@ -19,7 +19,8 @@ Following features and tools are supported:
 
 * 🐟 Fish Shell
 * 📷 AzCopy
-* ⚙️  Direnv
+* 🔐 Bitwarden
+* ⚙️ Direnv
 * ⛵️ Helm
 * 🛠  JQ
```

Comment:
well, if we look for superflous blanks: here is another one 😉

Reply:
Fixed